docs/root/intro/version_history.rst

```diff
@@ -66,6 +66,7 @@ Version history
   disables the use of deprecated extension names.
 * runtime: integer values may now be parsed as booleans.
 * sds: added :ref:`GenericSecret <envoy_api_msg_auth.GenericSecret>` to support secret of generic type.
+* sds: added :ref:`certificate rotation <xds_certificate_rotation>` support for xDS gRPC connection.
```

Comment:
This should work for any SDS setup, right? Meaning this can be used for a standard TLS listener cert rotation and is not specific to xDS? This is a feature request that comes up a ton so can we a) make it clear that this is also supported, and maybe also b) mention this somewhere in https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/security/ssl with maybe a link to the example you created?